src/snowflake/snowpark/modin/plugin/_internal/aggregation_utils.py

```diff
@@ -826,6 +826,66 @@ def aggregate_with_ordered_dataframe(
     return agg_ordered_dataframe
 
 
+def convert_named_agg_func_art_to_include_snowflake_quoted_identifier(
+    internal_frame: InternalFrame,
+    agg_func: dict[Hashable, dict[Hashable, AggFuncWithLabel]],
```

Comment:
Is `agg_func` the result of `extract_validate_and_try_convert_named_aggs_from_kwargs`? The name of this function is a little misleading since it's not actually a `NamedAgg` object at this point.